src/Latte/Compiler/Compiler.php

```diff
@@ -180,9 +206,11 @@ public function compile(array $tokens, string $className): string
 			$members[] = "\n\tfunction $name($method[arguments])\n\t{\n" . ($method['body'] ? "\t\t$method[body]\n" : '') . "\t}";
 		}
 
+		$useNamespaces = constant("{$this->templateCLass}::USE_NAMESPACES");
+
 		return "<?php\n"
-			. "use Latte\\Runtime as LR;\n\n"
-			. "class $className extends Latte\\Runtime\\Template\n{\n"
+			. ($useNamespaces ? Helpers::composeUseNamespaces($useNamespaces) . "\n" : '')
```

Comment:
`PHP_EOL` instead of `"\n"` could be more robust here.

Reply:
PHP_EOL is strictly worse; you cannot have a random line with inconsistent ending